connector_redmine/models/redmine_backend.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.exceptions import Warning
+from openerp.tools.translate import _
+from openerp.addons.connector.connector import ConnectorEnvironment
+from ..session import RedmineConnectorSession
```

Comment:
for the external dependencies you might want to use the following pattern: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror